CMakeLists.txt

```
@@ -1087,9 +1087,23 @@ add_library_variant(
 
 configure_file(
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake/multilib.yaml.in
-    ${CMAKE_CURRENT_BINARY_DIR}/llvm/${TARGET_LIBRARIES_DIR}/multilib.yaml
+    ${CMAKE_CURRENT_BINARY_DIR}/multilib.yaml.in.manual
```

The filename could be made more obvious.
```suggestion
    ${CMAKE_CURRENT_BINARY_DIR}/multilib-without-fpus.yaml
```

---

Fine, although if we later add more autogeneration we'll need to rename it :-)